### ISSUE TITLE
RStudio Server Help Search Field

### ISSUE THREAD

**Issue by bounlu** [rstudio/rstudio#14259]
### System details

    RStudio Edition : Server
    RStudio Version : 2023.03.0 Build 386
    OS Version      : Ubuntu 22.04.3 LTS
    R Version       : 4.3.2

### Steps to reproduce the problem

Install RStudio Server to Ubuntu and access via browser. Use Help panel for any function.

### Describe the problem in detail

Help panel search field does not have arrows to keep searching for Next and Previous item in the Help text. The search field loses focus after the first time Enter is pressed.

### Describe the behavior you expected

It should keep searching for the next found items in the Help text when pressing Enter and wrap around as in the Desktop version.

<img width="862" alt="Screenshot 2024-02-13 at 16 27 03" src="https://github.com/rstudio/rstudio/assets/17899927/5d213d90-c801-44a4-aab1-48db7f4ffafc">

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.posit.co/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

```[tasklist]
### Tasks
```

**Comment by ronblum** [comment]
@bounlu Thank you for raising the issue! I can't reproduce it, though—as soon as I start typing anything in the search box, the arrows appear. What operating system and browser are you using, and did this only start recently?

<img width="390" alt="arrows" src="https://github.com/rstudio/rstudio/assets/31009952/11fddd88-e8f4-4d71-ba8e-138bcf155d04">

**Comment by bounlu** [comment]
I'm using the latest version of Firefox. The RStudio Server is running on Ubuntu Server 22. This has been there for a very long time, I just reported though. It works as expected on my Desktop version on Mac.

**Comment by bounlu** [comment]
I confirm that this issue is specific to Firefox, as it works fine with Google Chrome.

**Comment by gtritchie** [comment]
Research note:  these next/prev buttons were intentionally disabled via following code. I'll see if this is still needed (this was done 11+ years ago).

https://github.com/rstudio/rstudio/blob/6a72329f859aaa6c56866b46de8a7bc7f0c03a5f/src/gwt/src/org/rstudio/studio/client/workbench/views/help/HelpPane.java#L797-L804

**Comment by gtritchie** [comment]
The same problem that led to this Firefox-specific behaviour still exists, so we can't simply remove this exclusion.
